frontend/src/CommentView.js

```diff
@@ -0,0 +1,58 @@
+// import { svg } from "leaflet";
+import React, { useState } from "react";
+import Collapsible from 'react-collapsible';
+
+import { ReactComponent as ThumbsUp } from "./thumbs-up.svg";
+import { ReactComponent as ThumbsDown } from "./thumbs-down.svg";
+
+function CommentViewHeader(props){
+    return (
+        <div>
+            <div style={{fontSize: 20}}>{props.pin.title}</div>
+            <div>{props.pin.description}</div>
+            <br></br>
+            <div>Comments:</div>
+        </div>
+    );
+}
+
+function CommentViewBody(props) {
+    const _comments = props.commentsData.map( (text, index) => {
+        return(
+            <Collapsible key={index} trigger = "▼/▲" open = 'true'>
+                {text.description} <br></br>
+                <ThumbsUp title="Upvotes" onClick={() => props.upVote(index)}/> {text.upvotes} 
+                <ThumbsDown title="Downvotes" onClick={() => props.downVote(index)}/> {text.downvotes} 
+            </Collapsible>
+        );
+    });
+    return <div>{_comments}</div>;
+}
+
+function CommentView(){
+    let pin = {title:'Study Bench', description:'Nice stone bench with shade from trees', upVotes:3, downVotes:4,pinType:'Study Location', color:'green'}
+    let commentz = [{description:"I go here all the time", downvotes: 10, upvotes: 50},{description:"I got my shoes dirty here, I don't like it", downvotes: 6, upvotes: 32}]
```

Comment:
What is commentz?

Reply:
That is the name for the json object I created to act as a filler for the backend comment data.

Reply:
Understood. Looks good to me!